config/config_training.yaml

```diff
@@ -0,0 +1,23 @@
+
+# Input config
+model_config: "config/config_models_test.tsv"
+dataset_config: "config/config_datasets_test.tsv" 
+run_feature_analysis: True # do you want to run feature analysis? 
+
+# Output directory
+results_dir: "results_test/"
+
+# Reference files
+gene_TSS500: "reference/RefSeqCurated.170308.bed.CollapsedGeneBounds.hg38.TSS500bp.bed" # TSS reference file
+chr_sizes: "reference/GRCh38_EBV.no_alt.chrom.sizes.tsv"
+ubiq_expr_genes: "reference/for_classifier_use_ubiquitous_expression_RefSeqCurated.170308.bed.CollapsedGeneBounds.hg38.TSS500bp.txt"
+crispr_dataset: "reference/EPCrisprBenchmark_ensemble_data_GRCh38.tsv.gz" # CRISPR dataset
+
+# Model parameters
+epsilon: .01
+
+### INTERNAL USE ONLY
+ABC_DIR_PATH: "ABC"
+# e2g path to prepend to reference and script files
+# Only relevant when using e2g as a submodule
+E2G_DIR_PATH: "/oak/stanford/groups/engreitz/Users/sheth/ENCODE_rE2G"
```

Comment:
We should try to avoid publishing our oak filepaths. Can we just use `.` here to represent current directory? (There's existing handling in the code to convert relative paths to absolute)